node/src/blockchain/test_utils.rs

```diff
@@ -335,3 +384,23 @@ pub fn all_chains() -> [Chain; 4] {
         Chain::Dev,
     ]
 }
+
+pub fn transport_error_code() -> u16 {
+    if cfg!(target_os = "windows") {
+        10061
+    } else if cfg!(target_os = "macos") {
+        61
+    } else if cfg!(target_os = "linux") {
+        111
+    } else {
+        0
+    }
+}
+
+pub fn transport_error_message() -> String {
+    if cfg!(target_os = "windows") {
+        "No connection could be made because the target machine actively refused it.".to_string()
+    } else {
+        "Connection refused".to_string()
+    }
+}
```

Comment:
I can't understand for what you need these two newly added functions, but that aside, they again feel to me as if they were't introduced only because you need them in some new test infrastructure. 

At least please make sure it touches the production code in no way.

Reply:
Yes this is never used in prod, just a single test :)